client/tokenized-payment-request/cart-api.js

```diff
@@ -0,0 +1,131 @@
+/* global jQuery */
+
+/**
+ * External dependencies
+ */
+import apiFetch from '@wordpress/api-fetch';
+import { applyFilters } from '@wordpress/hooks';
+
+export default class PaymentRequestCartApi {
+	// Used on product pages to interact with an anonymous cart.
+	// This anonymous cart is separate from the customer's cart, which might contain additional products.
+	// This functionality is also useful to calculate product/shipping pricing (and shipping needs)
+	// for compatibility scenarios with other plugins (like WC Bookings, Product Add-Ons, WC Deposits, etc.).
+	cartRequestHeaders = {};
+
+	async placeOrder( paymentData ) {
+		return await apiFetch( {
+			method: 'POST',
+			path: '/wc/store/v1/checkout',
+			credentials: 'omit',
+			headers: {
+				'X-WC-Payments-prb-request': true,
```

Comment:
> In regards to WCPayments & WooPayments - there are still a bunch of items that refer the old name

I think most of our enqueued global variables will be prefixed with `wcpay...`, `WCPayAPI` still exists, and of course our gateway slug is still `woocommerce_payments`. These are remnants of antique code, but I don't think there's really a directive here and even if we future-proof the header name here to use the current plugin name rather than the old one, ultimately we'll be the only ones using this header at all, as I don't believe its intention is to be user-facing. This one's up to you.